<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_olymp_medal</class>
 <widget class="QDialog" name="form_olymp_medal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font-family: 'Montserrat', sans-serif;
	font-size: 14px;
	color: rgb(71, 74, 81);
}

QDialog{
	background-color: rgb(255, 255, 255);
}

QLineEdit{
	border: 1px solid rgb(0, 188, 188);
	border-radius: 3px;
}

QPushButton{
	height: 20px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QPushButton:hover{
	height: 75px;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;
	font-size: 11px;
	text-transform: uppercase;
	font-weight: 600;
	color: rgb(205, 92, 92);
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:pressed{
	background-color: rgba(205, 92, 92, 1);
	color: white;
	border:2px solid;
	border: 1px solid rgba(205, 92, 92, 1) ;
	border-radius: 5px;
	font-family: 'Montserrat', sans-serif;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Medal data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID part country:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gold:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Silver:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bronze:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Count medal:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ID olymp:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>23</number>
     </property>
     <property name="leftMargin">
      <number>23</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>23</number>
     </property>
     <item>
      <widget class="QPushButton" name="but_save">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_can">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
